doc/guides/09-run_returning_interactions.md

```diff
@@ -0,0 +1,54 @@
+# Run Returning Interactions
+
+### Example: Reseting Niiwin App
+
+```ruby
+module Niiwin
```

Comment:
We don't want any connection to Niiwin - feel free to turn this into a simpler generic example